bin/physlr-make

```diff
@@ -177,7 +177,7 @@ help:
 	@echo "	min_component_size		minimum number of barcodes in a backbone [50]."
 	@echo "	minimum_barcode_multiplicity	minimum number of minimizers per barcode [10]."
 	@echo "	maximum_barcode_multiplicity	maximum number of minimizers per barcode [5000]."
-	@echo "	minimizer_multiplicity		maximum minimizer multiplicity [100]. Specify x to use the Q3 + IQR of the minimizer multiplicity distribution instead."
+	@echo "	minimizer_multiplicity		Available option is x. Specify x to use the Q3 + IQR of the minimizer multiplicity distribution instead of the repetitive k-mer threshold."
```

Comment:
So here are you saying that you can only specify `x`? That's the only valid option? 
If that's the case, it would make more sense to make it more boolean (1 for on, 0 for off, say)

Reply:
I've decided to allow the user to specify their own minimizer_multiplicity